ttnn/cpp/ttnn/operations/full/device/full_device_operation.cpp

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "full_device_operation.hpp"
+
+#include "ttnn/tensor/tensor.hpp"
+
+namespace ttnn::operations::full {
+void FullOperation::validate_inputs(
+    const operation_attributes_t& operation_attributes, const tensor_args_t& tensor_args) {
+    auto any = tensor_args.any;
```

Comment:
why `any`? this name is very ambiguos

Reply:
Since this operation doesn’t require any tensors as input, we need to pass `any` tensor to retrieve the `device`, `dtype`, `layout`, and `memory_config`, if necessary. This is similar to how the moreh_arange operation is handled